### PR TITLE
feat(patterns/passwordinput): add code and associated documentation

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Add new **Password Input** component

GitHub issue number or Jira issue URL: https://jira.adeo.com/browse/MZC-3

### CODE REVIEW

File: src/docs/Components/Form/PasswordInput/index.mdx [hunk 1]
```diff
@@ -17,7 +17,7 @@ status:
 
 > Password input is a necessary security feature for logging in or registering on a website.
 
-Place preview here !
+<Preview path="default" nude />
```
Comment (caqueste) [inline src/docs/Components/Form/PasswordInput/index.mdx:20]: @tiloyi Can you prefill the input with a password? For example "Password1234".

**Review by caqueste — CHANGES_REQUESTED** [review]
@tiloyi Can you please add a preview, with two inputs at the end of the page?
For the first one, the password will be hidden, and for the second one, the password will be displayed.

File: src/docs/Components/Form/PasswordInput/index.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/Components/Form/PasswordInput/index.mdx:20]: Fixed

**Review by caqueste — CHANGES_REQUESTED** [review]
Sorry @tiloyi, can you please add this preview at the end of the usage page "<Preview path="all-cases" nude />" ?
Thanks

**Review by caqueste — APPROVED** [review]

**Pull request merged**